item-spec/item-spec.md

```diff
@@ -193,11 +193,16 @@ This happens where there is not a clear official option, or where STAC uses an o
 | Type         | Description                                                                                                                                                                                                                                                                                  |
 | ------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | self         | STRONGLY RECOMMENDED. *Absolute* URL to the Item if it is available at a public URL. This is particularly useful when in a download package that includes metadata, so that the downstream user can know where the data has come from.                                                       |
-| root         | URL to the root STAC entity (Catalog or Collection).                                                                                                                                                                                                                                         |
-| parent       | URL to the parent STAC entity (Catalog or Collection).                                                                                                                                                                                                                                       |
+| root         | URL to the root STAC entity (Catalog or Collection). ONLY ONE link with `root` relationship to STAC entity is allowed.                                                                                                                                                                       |
+| parent       | URL to the parent STAC entity (Catalog or Collection). ONLY ONE link with `parent` relationship to STAC entity is allowed.                                                                                                                                                                   |
```

Comment:
I think we need to say here that a link to a JSON-version of a single STAC-entity is allowed. Multiple root or parent links can occur if they point to different encodings or language versions, but they should all point to the same entitiy (e.g. identified by the ID)